VideoIntelligence/src/V1/Gapic/VideoIntelligenceServiceGapicClient.php

```diff
@@ -251,10 +251,7 @@ public function __construct(array $options = [])
      * ```
      * $videoIntelligenceServiceClient = new VideoIntelligenceServiceClient();
      * try {
-     *     $inputUri = 'gs://cloud-samples-data/video/cat.mp4';
-     *     $featuresElement = Feature::LABEL_DETECTION;
-     *     $features = [$featuresElement];
-     *     $operationResponse = $videoIntelligenceServiceClient->annotateVideo(['inputUri' => $inputUri, 'features' => $features]);
+     *     $operationResponse = $videoIntelligenceServiceClient->annotateVideo();
```

Comment:
Hmm, this documentation is really helpful. Is there any way we can keep this? I'm thinking this was added with post-processing somehow.

Reply:
Addressed in https://github.com/googleapis/google-cloud-php/pull/5044